src/app/files.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+var EventManager = require('../lib/eventManager')
+
+function Files (storage) {
+  var event = new EventManager()
+  this.event = event
+  var readonly = {}
+
+  this.exists = function (path) {
+    // NOTE: ignore the config file
+    if (path == '.browser-solidity.json') {
```

Comment:
In the future I would move it outside this and change the config API to support that (i.e. catch events modifying this file). That way the user could both change in the editor as well as in the UI.